docs/20_setup.md

```diff
@@ -59,14 +72,43 @@ chmod +x /tmp/miactl
 mv /tmp/miactl /usr/local/bin
 ```
 
-### Docker
+If the `mv` command replies with a `Permission denied`, retry it as root:
+```sh
+sudo mv /tmp/miactl /usr/local/bin
+```
+
+#### Docker
 
 If you want to run the cli in its environment or you want to test the cli you can use the Docker image:
 
 ```sh
 docker run ghr.io/mia-platform/miactl@0.7.0 miactl
 ```
 
+### Windows
+
+`miactl` is not directly compatible with Windows, even if you have Go installed: compilation on this OS is not possible due to current technical restrictions.
+
+However, it is still possible to use `miactl` with Windows Subsystem for Linux (WSL), as explained here below.
+
+#### Installation of WSL
+
+If you don't have WSL on your system, follow the [official guide](https://learn.microsoft.com/en-us/windows/wsl/install) to get it.
+
+Once WSL is installed, to open a Linux bash terminal, press Start+R, enter `bash` in the text box and press OK.
+
+#### Install `miactl`
+
+You can now install `miactl  with any of the methods explained above for Linux, we suggest the [binary installation](#binary-download) since it's the most straightforward.
```

Comment:
```suggestion
You can now install `miactl`  with any of the methods explained above for Linux, we suggest the [binary installation](#binary-download) since it's the most straightforward.
```